content/agent/faq/send-logs-and-configs-to-datadog-via-flare-command.md

```diff
@@ -1,58 +1,65 @@
 ---
-title: Send logs and configs to Datadog via flare command
+title: Send logs and configs to Datadog via flare
 kind: faq
 further_reading:
 - link: "/agent/"
   tag: "Documentation"
   text: Learn more about the Datadog Agent
 ---
 
-## Linux and Mac OS X
+## Overview
 
-If you are running the 5.3 version of the agent, you're able to send all necessary troubleshooting information to our Support Team, with one command!
+If you are running the 5.3 version of the agent (or higher), you're able to send all necessary troubleshooting information to our Support Team, with one flare command!
 
-First, find the datadog-agent command:
+Flare gathers all of the agent's configuration files and logs, remove sensitive information like passwords, passwords in a URI, API keys, Proxy credentials, and SNMP community strings... and upload it to our server as an archive. Since the Datadog Agent is completely open source, [check out the code for this at any time](https://github.com/DataDog/dd-agent/blob/master/utils/flare.py).  
```

Comment:
`flare` gathers all of the agent's configuration files and logs, removes sensitive information including passwords, API keys, Proxy credentials, and SNMP community strings and uploads it to Datadog as an archive. Since the Datadog Agent is completely open source, you can [verify the code's behavior](https://github.com/DataDog/dd-agent/blob/master/utils/flare.py).

Reply:
👍